### ISSUE TITLE
Installation of amplify-ui 2.0 with Angular 12.2.x fails

### ISSUE THREAD

**Issue by fbu-nlstbv** [aws-amplify/amplify-ui#806]
It's not possible to install Amplify UI 2.0 with Angular 12.2.x, see the following command line output:

```
PS C:\[...]\todo-amp\todo-app-amplify> npm i --save @aws-amplify/ui-angular
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: todo-amp@0.0.0
npm ERR! Found: @angular/common@12.2.13
npm ERR! node_modules/@angular/common
npm ERR!   @angular/common@"~12.2.13" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @angular/common@"^11.2.4" from @aws-amplify/ui-angular@2.0.0
npm ERR! node_modules/@aws-amplify/ui-angular
npm ERR!   @aws-amplify/ui-angular@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!

```

**Comment by hvergara** [comment]
Hey @fbu-nlstbv!

Thanks for reporting it, we'll take a look and provide a solution shortly

**Comment by ericclemmons** [comment]
Ah, let's update our Angular example to use v12:

https://github.com/aws-amplify/amplify-ui/blob/83d2f78b7225e1923c40c8cc55335a5ae5cf9822/examples/angular/package.json#L29-L30

**Comment by ericclemmons** [comment]
@fbu-nlstbv We have a quick fix here:
> https://github.com/aws-amplify/amplify-ui/pull/807

Once the e2e tests pass, we'll merge it & get it published as `@aws-amplify/ui-angular@next`.

Until then, does `--legacy-peer-deps` unblock you?

**Comment by fbu-nlstbv** [comment]
Yes, "--legacy-peer-deps" works as a fix for now. Thanks for your fast response and I am looking forward for your fixed solution.

**Comment by ericclemmons** [comment]
@fbu-nlstbv Can you validate that it works with `@aws-amplify/ui-angular@next`, too? 🙏 

Re-opening until then, so that it can be officially released in https://github.com/aws-amplify/amplify-ui/pull/792.

**Comment by fbu-nlstbv** [comment]
Sorry for quickly closing the issue. Yes, `npm install --save @aws-amplify/ui-angular@next` works successfully with a Angular 12.2.x dependency in the package.json.

**Comment by wlee221** [comment]
Great! I'll close this issue then. We just released this patch to `@aws-amplify/ui-angular@2.0.1`, so you can now switch back to `latest`.
